Godeps/_workspace/src/github.com/GoogleCloudPlatform/kubernetes/pkg/api/types.go

```diff
@@ -2162,45 +2162,49 @@ type RangeAllocation struct {
 // SecurityContextConstraints governs the ability to make requests that affect the SecurityContext
 // that will be applied to a container.
 type SecurityContextConstraints struct {
-	TypeMeta   `json:",inline"`
-	ObjectMeta `json:"metadata,omitempty"`
+	TypeMeta
+	ObjectMeta
 
 	// AllowPrivilegedContainer determines if a container can request to be run as privileged.
-	AllowPrivilegedContainer bool `json:"allowPrivilegedContainer,omitempty" description:"allow containers to run as privileged"`
+	AllowPrivilegedContainer bool
 	// AllowedCapabilities is a list of capabilities that can be requested to add to the container.
-	AllowedCapabilities []Capability `json:"allowedCapabilities,omitempty" description:"capabilities that are allowed to be added"`
+	AllowedCapabilities []Capability
 	// AllowHostDirVolumePlugin determines if the policy allow containers to use the HostDir volume plugin
-	AllowHostDirVolumePlugin bool `json:"allowHostDirVolumePlugin,omitempty" description:"allow the use of the host dir volume plugin"`
+	AllowHostDirVolumePlugin bool
```

Comment:
Hurry - want to get this merged!

On Wed, Jul 22, 2015 at 1:27 PM, Paul Morie <notifications@github.com>
wrote:

> In
> Godeps/_workspace/src/github.com/GoogleCloudPlatform/kubernetes/pkg/api/types.go
> <https://github.com/openshift/origin/pull/3496#discussion_r35239350>:
>
> >  	// AllowHostDirVolumePlugin determines if the policy allow containers to use the HostDir volume plugin
> > -	AllowHostDirVolumePlugin bool `json:"allowHostDirVolumePlugin,omitempty" description:"allow the use of the host dir volume plugin"`
> > +	AllowHostDirVolumePlugin bool
>
> You might consider making the comment changes in another PR -- I had one
> that did this, which is closed, but this seems like something someone might
> just go and do and then could fall out as a rebase problem in the future.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/openshift/origin/pull/3496/files#r35239350>.
>



-- 
Clayton Coleman | Lead Engineer, OpenShift